tick/optim/model/src/coxreg_partial_lik.h

```diff
@@ -9,8 +9,7 @@
 
 #include "model.h"
 
-
-class ModelCoxRegPartialLik : public Model {
+class DLL_PUBLIC ModelCoxRegPartialLik : public Model {
```

Comment:
This is the same for all model classes except "Hawkes" ones. I've seen that for Prox you've only done it with ProxZero and its ancestors.